### PR TITLE
CDAP-5574 dont log unnknown program type for custom action and webapp

### PR DESCRIPTION
**Pull request by gsps1**
JIRA https://issues.cask.co/browse/CDAP-5574
Build http://builds.cask.co/browse/CDAP-DUT4119-1

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java [hunk 1]
```diff
@@ -772,6 +772,12 @@ private ProgramId retrieveProgramIdForRunRecord(ProgramType programType, String
               }
             }
             break;
+          case CUSTOM_ACTION:
```
Comment (chtyim) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java:775]: You can collapse it into

``` java
case CUSTOM_ACTION:
case WEBAPP:
//no-op
break;
```

Comment (gsps1) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java:775]: done

**Comment by chtyim** [comment]
LGTM

**Comment by gsps1** [comment]
build green, merging.

**Pull request merged**